charts/newrelic-infrastructure/README.md

```diff
@@ -44,13 +44,15 @@ This chart will deploy the New Relic Infrastructure agent as a Daemonset.
 | `etcdEndpointUrl`              | Explicitly sets the etcd component url.                                                                                                                                                                                                           |                                 |
 | `apiServerSecurePort`          | Set to query the API Server over a secure port.                                                                                                                                                                                                   |                                 |
-| `apiServerEndpointUrl`         | Explicitly sets the api server componenturl.                                                                                                                                                                                                      |                                 |
+| `apiServerEndpointUrl`         | Explicitly sets the api server component url.                                                                                                                                                                                                     |                                 |
 | `schedulerEndpointUrl`         | Explicitly sets the scheduler component url.                                                                                                                                                                                                      |                                 |
 | `controllerManagerEndpointUrl` | Explicitly sets the controller manager component url.                                                                                                                                                                                             |                                 |
 | `eventQueueDepth`              | Increases the in-memory cache of the agent to accommodate for more samples at a time. | |
 | `enableProcessMetrics`         | Enables the sending of process metrics to New Relic.  | `false` |
 | `global.nrStaging` - `nrStaging` | Send data to staging (requires a staging license key). | `false` |
-| `discoveryCacheTTL`            | Duration since the discovered endpoints are stored in the cache until they expire. Valid time units: 'ns', 'us', 'ms', 's', 'm', 'h' | `1h` | |
+| `discoveryCacheTTL`            | Duration since the discovered endpoints are stored in the cache until they expire. Valid time units: 'ns', 'us', 'ms', 's', 'm', 'h' | `1h` |
+| `openshift` |  Enables OpenShift configuration | `false` |
+| `caBundleFile` | Path to custom CA Bundle file | |
```

Comment:
caBundleFile is added only to values, but if I am not wrong it is not used in the template, is tis a typo?